sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -64,7 +64,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       val queryCoalesce = sql("select coalesce(null, 1, '1') from src ")
 
       checkAnswer(queryCaseWhen, Row("1.0") :: Nil)
-      checkAnswer(queryCoalesce, Row("1") :: Nil)
+      checkAnswer(queryCoalesce, Row("2") :: Nil)
```

Comment:
```suggestion
      checkAnswer(queryCoalesce, Row("2") :: Nil)
```

Reply:
In worst case (e.g., author is inactive but PR is ready .. ?), we could push an empty commit to their branch to retrigger .. of course this is an ugly way though